### PR TITLE
ci: add npm audit

### PR DESCRIPTION
**Pull request by stringhandler**
Add npm audit to CI check.

Also updated some dependencies using `npm audit fix`

### CODE REVIEW

**Review by delta1 — DISMISSED** [review]
yolo

File: applications/launchpad/gui-vue/package.json [hunk 1]
```diff
@@ -20,13 +20,13 @@
   },
   "devDependencies": {
     "@tauri-apps/cli": "^1.0.0-beta.10",
-    "eslint-plugin-vue": "^7.0.0",
     "@vue/cli-plugin-babel": "~4.5.0",
```
Comment (Cifko) [inline applications/launchpad/gui-vue/package.json]: This should be ^5.0.4

File: applications/launchpad/gui-vue/package.json [hunk 2]
```diff
@@ -20,13 +20,13 @@
...
-    "@vue/cli-plugin-eslint": "~4.5.0",
-    "@vue/cli-service": "~4.5.0",
+    "@vue/cli-plugin-eslint": "^5.0.4",
+    "@vue/cli-service": "^5.0.4",
     "@vue/compiler-sfc": "^3.0.0",
     "babel-eslint": "^10.1.0",
-    "eslint": "^6.7.2"
+    "eslint": "^6.7.2",
```
Comment (Cifko) [inline applications/launchpad/gui-vue/package.json]: And this should be ^7.0.0

File: applications/launchpad/gui-vue/package.json (hunk 1 above)
Comment (stringhandler) [inline applications/launchpad/gui-vue/package.json]:
```suggestion
    "@vue/cli-plugin-babel": "^5.4.0",
```

File: applications/launchpad/gui-vue/package.json (hunk 2 above)
Comment (stringhandler) [inline applications/launchpad/gui-vue/package.json]:
```suggestion
    "eslint": "^7.0.0",
```

**Review by sdbondi — APPROVED** [review]

**Pull request merged**